swarm/metrics/flags.go

```diff
@@ -91,7 +92,10 @@ func Setup(ctx *cli.Context) {
 		)
 
 		// Start system runtime metrics collection
-		go gethmetrics.CollectProcessMetrics(2 * time.Second)
```

Comment:
No need to collect them every 2sec., for now I think we are happy with our 10sec. resolution.